docs/index.md

```diff
@@ -0,0 +1,55 @@
+
+Get ready to embark on an unforgettable adventure with Wapuugotchi, where your daily routines transform into exciting quests and where personalized Wapuu characters brighten your every moment. Join us in this one-of-a-kind project that promises to revolutionize the way you engage with technology and bring a smile to your face every step of the way. Welcome to Wapuugotchi, your personalized world of joy, fun, and endless possibilities.
+
+## Install plugin
+The installation of the plugin is very straightforward. You can download it from the WordPress.org Plugin Directory as zip file or you can install it directly from your WordPress over the default plugin area. For that, you click on "Plugins" in the sidebar navigation and then on "Add New". You will find Wapuugotchi over the search.
+
+## Dashboard
+After the installation, you will see your Wapuu in the Dashboard area on the bottom right corner. Your new friend will show you notifications here.
+
+![Wapuu in the dashboard](assets/images/wapuugotchi_on_dashboard.png)
+
+## Sidebar navigation
+You wil find *WapuuGotchi* in the sidebar navigation menu of WordPress. When you click on it, you will be redirected to the *Customizer* by default. There is another page you can access, the *Quest Log*.
+
+![Sidebar navigation options](assets/images/sidebar_navigation.png)
+
+## Configure your Wapuu
+To configure your little pal, the plugin provides you with a boutique full of items which you already own or which can be exchanged for precious pearls. To enter the boutique click on *WapuuGotchi* in the sidebar navigation. You should see the customizer now.
+
+![Wapuu Customizer](assets/images/customizer.png)
+
+Here in the customizer, you have different types of wearables, which you can use to dress your Wapuu. First of all you have some basic shapes, which differ in form and color. There is a section with things which the Wapuu can hold, like orbs or guitars. Furthermore you can give your little fellow a hat, a haircut or even a crown. Then you have a category with general accessiores like a superhero cape, a baguette or smart-looking glasses. Your Wapuu can also wear a sweater or a t-shirt. And finally you can also dress him with different shoes, which suit to the rest of their outfit.
+
+Some of the items are locked in the beginning and can be unlocked through pearls.
+
+![A locked item in the Customizer](assets/images/customizer_locked_item.png)
+
+When you click on a locked wearable, a modal will popup and ask you if you want to buy this item. On the top right you can find an information icon. When you hover over that item, you will see more information about the currently shown Wapuu. If you click on *OK", you confirm your purchase and therefore unlock the item.
```

Comment:
The *OK" seems wrong, either *OK* or "OK" should be correct.

Reply:
Ty, fixed!